cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java

```diff
@@ -211,6 +212,7 @@ protected void configure() {
                       .build(DatasetDefinitionRegistryFactory.class));
             // CDAP-5954 Upgrade tool does not need to record lineage and metadata changes for now.
             bind(LineageWriter.class).to(NoOpLineageWriter.class);
+            bind(FieldLineageWriter.class).to(NoOpLineageWriter.class);
```

Comment:
Hmmm. I find this dangerous. In the next release, it may need to be able to write lineage, and then it will use no-op? Is there a harm in binding it to the actual implementation? I find that more future-proof.